modules/nf-core/universc/mkfastq/main.nf

```diff
@@ -0,0 +1,46 @@
+process UNIVERSC_CELLRANGER_OS_MKFASTQ {
```

Comment:
```suggestion
process UNIVERSC_MKFASTQ {
```
Is there a reason to break the standard here?

Reply:
What's the difference between this module and the `CELLRANGER_MKFASTQ` module?

Reply:
I've updated all process names to match this consistent format. The references must be generated with the older version of Cell Ranger. The demultiplex function "cellranger mkfastq" is also supported by the open-source Cell Ranger v3.0.2 but it is not essential for UniverSC to function. If it is unnecessary, I'll remove universc/cellranger_os_mkfastq from the repository.